MetaMask/metamask-extension
# Replaced ActionableMessage component in srp-input

## Explanation


- This PR replaces all `ActionableMessage` components in `ui/components/app/srp-input/srp-input.js` with `BannerAlert` component .
- This PR is a part of #19528 

## Screenshots/Screencaps



### Before


[srp-input before](https://github.com/MetaMask/metamask-extension/assets/46365255/96b01300-9b41-4587-b68f-deb6900e7550)

### After


[srp-input-after](https://github.com/MetaMask/metamask-extension/assets/46365255/a3b6b93d-cfdd-4693-aa22-df4082d5eb91)

## Manual Testing Steps


- Check the Storybook build of `srp-input` component
- Create new development build 
## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
